pkg/ccl/changefeedccl/event_processing.go

```diff
@@ -174,6 +218,8 @@ func newKVEventToRowConsumer(
 	expr execinfrapb.Expression,
 	knobs TestingKnobs,
 	topicNamer *TopicNamer,
+	pacer *admission.Pacer,
+	pacerLogEvery *log.EveryN,
```

Comment:
you could use a single (global) everyN object for all pacers -- I doubt you need it per consumer.

Reply:
Done.